main_renderer.js

```diff
@@ -57,10 +57,13 @@ $(document).ready(() => {
                     .append($('<br />'));
             }
 
+            let _class = 'table-active';
+            if (firstItemSelected) _class = '';
+
             $('#documentList')
                 .append($('<tr/>')
                     .attr('onclick', `preview("${asset.assetID}")`)
-                    .attr('class', firstItemSelected ? '' : 'table-active')
+                    .attr('class', _class)
```

Comment:
I preferred the previous approach, it's way easier to read and you are not using `_class` anywhere else.

Reply:
The previous approach was giving lint errors. But I managed to make it pass by adding newlines. The lint has the `multiline-ternary` rule on.